subiquity/server/controllers/network.py

```diff
@@ -236,6 +237,15 @@ async def GET(self) -> NetworkStatus:
         )
 
     async def configured(self):
+        # There may be some instances in which the network controller
+        # is interactive but is only ever marked configured (through a
+        # POST to either /network or /meta/mark_configured) and is never
+        # interacted with otherwise (via GET) such that we don't disable
+        # interfaces which don't have a global IP.
+        if self.interactive() and not self.view_shown:
+            self.view_shown = True
```

Comment:
The name of the variable does not reflect what happens (anymore?) does it?

Reply:
I should probably drop modifying that variable in that clause. It was to avoid re-invoking the `update_inital_config` logic if the client later did a `GET` but that should probably be fine.